tests/test_app/test_data_store_new.py

```diff
@@ -555,3 +557,36 @@ def test_load_record_from_json():
         assert Id == "some.json"
         assert data_ == expected
         assert compl == True
+
+
+def test_write_read_not_completed(nc_dstore):
+    nc_dstore.drop_not_completed()
+    assert len(nc_dstore.not_completed) == 0
+    nc = NotCompleted("ERROR", "test", "for tracing", source="blah")
+    writer = io_app.write_seqs(data_store=nc_dstore)
+    writer.main(nc, identifier="blah")
+    assert len(nc_dstore.not_completed) == 1
+    reader = io_app.load_aligned()
+    got = reader(nc_dstore.not_completed[0])
```

Comment:
@GavinHuttley 
There is problem here. When reads NotCompleted, it is not equal to the written Notcompleted.